recipes/maggma/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "maggma" %}
+{% set version = "0.24.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/maggma-{{ version }}.tar.gz
+  sha256: c6349bd1419a974803aaa8abf5a5b0d0f5c94a1c61e86def881e79490e32c085
+  patches:
+    - msgpack.patch
+
+build:
+  number: 0
+  skip: true  # [win or py<37]
```

Comment:
Any reason for not being noarch?

Reply:
Only that it depends on `pynng` which is only available for linux and mac os x

Reply:
That's fine, the Windows CI can be ignored if it's just missing deps